src/Cache/RedisSentinelStore.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace TusPhp\Cache;
+
+use Carbon\Carbon;
+use TusPhp\Config;
+use Predis\Client as RedisClient;
+
+class RedisSentinelStore extends AbstractCache
+{
+    /** @var RedisClient */
+    protected $redis;
+
+    /**
+     * RedisStore constructor.
+     *
+     * @param array $options
+     */
+    public function __construct(array $params = [], array $options = [])
+    {
+        $options = empty($options) ? Config::get('sentinel.options') : $options;
+        $params = empty($params) ? Config::get('sentinel.params') : $params;
```

Comment:
Can we add an example of how sentinel options/params looks like, probably in the README?